src/Perception/include/BipedalLocomotion/Perception/Capture/RealSense.h

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
There was a discussion on this regarding removing the meta-data repo-wide because of its redundancy. See https://github.com/dic-iit/bipedal-locomotion-framework/pull/165#discussion_r545022152